controls/pcidss_ocp4.yml

```diff
@@ -35,7 +37,10 @@ controls:
       cardholder data environment and other networks, including any wireless networks
     levels:
     - base
-    notes: ''
+    notes: |-
+      While there are ways and tools where the OpenShift Container Platform could
+      help address this control, covering the scope of a deployment with the application
+      specific connections is out of scope.
```

Comment:
Would it make sense to recommend stackrox/ACS here?

Reply:
done